GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2454,7 +2454,7 @@ private void RevisionInfo_CommandClicked(object sender, ResourceManager.CommandE
 
                     if (found)
                     {
-                        found = RevisionGrid.SetSelectedRevision(revision);
+                        found = RevisionGrid.SetSelectedRevision(revision, showNoRevisionMsg: !AppSettings.ShowFirstParent);
```

Comment:
`RevisionInfo_CommandClicked()` is called when links in the Commit tab is clicked.